style50/core.py

```diff
@@ -167,18 +167,28 @@ def _check(self, file):
         _, extension = os.path.splitext(file)
         try:
             check = self.extension_map[extension[1:]]
+
             with open(file) as f:
                 code = f.read()
+
         except (OSError, IOError) as e:
             if e.errno == errno.ENOENT:
                 raise Error("file \"{}\" not found".format(file))
             else:
                 raise
-        except KeyError:
+        except (KeyError, IndexError):
```

Comment:
They `IndexError` is actually probably not strictly necessary. I was going for accounting for the case that `extension` was fewer than 1 characters, but it seems that `""[1:]` returns the empty string instead of raising an `IndexError` (which is what I'd have expected)